### PR TITLE
{bio}[intel/2016a] EggLib v2.1.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

requires https://github.com/hpcugent/easybuild-easyblocks/pull/811

### CODE REVIEW

**Comment by boegel** [comment]
This works, but maybe there's too much trickery going on in here, which should be handled via an easyblock instead...

@wpoely86: please review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5db9d2c723a1fade364d for a full test report.

File: easybuild/easyconfigs/e/EggLib/EggLib-2.1.10-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [
+    '%(namelower)s-cpp-%(version)s.tar.gz',
+    '%(namelower)s-py-%(version)s.tar.gz',
+]
+
+dependencies = [
+    ('Python', '2.7.11'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/e/EggLib/EggLib-2.1.10-intel-2015b.eb]: shouldn't we stick to `.10` to avoid dep hell?

**Comment by wpoely86** [comment]
This one feels dirty.

File: easybuild/easyconfigs/e/EggLib/EggLib-2.1.10-intel-2015b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/e/EggLib/EggLib-2.1.10-intel-2015b.eb]: We'll have to move to `.11` sooner or later. I already did for DOLFIN, for example.

And we're really close to moving on to `intel/2016a` now, so jumping to Python 2.7.11 is going to happen anyway.

**Comment by boegel** [comment]
@wpoely86: it does, it needs an easyblock for sure

File: easybuild/easyconfigs/e/EggLib/EggLib-2.1.10-intel-2015b.eb (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyconfigs/e/EggLib/EggLib-2.1.10-intel-2015b.eb]: for `2016a` sure, but I would stick to `.10` for `intel2015b`

**Comment by boegel** [comment]
@wpoely86: updated, now requires https://github.com/hpcugent/easybuild-easyblocks/pull/811

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c934b4c0c6a8d0c90b8c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d31c908948c0441c3275 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
All cleaned up, good to go, thanks for the review @wpoely86!

**Pull request merged**